Modules/clinic/cmathmodule.c.h

```diff
@@ -644,7 +644,7 @@ PyDoc_STRVAR(cmath_log__doc__,
 "\n"
 "log(z[, base]) -> the logarithm of z to the given base.\n"
 "\n"
-"If the base not specified, returns the natural logarithm (base e) of z.");
+"If the base not specified, returns the natural logarithm of z.");
```

Comment:
I think the "base e" should be kept here; I think it's genuinely useful to those reading the docs who might otherwise expect the base to be 10 (or 2), and who haven't encountered the term "natural logarithm" before.

I understand that the goal of this PR is to resolve a theoretical ambiguity between the `float` value `math.e` and Euler's number, but I honestly don't think that ambiguity is likely to confuse people in practice.

For correctness, I'd suggest using `*e*` throughout (i.e., italicised normal text) when referring to Euler's number, and code-style formatting (using backticks)  when referring to `math.e`. But I think this blanket removal of references to `e` is more harmful than helpful.

Reply:
> I'd suggest using *e* throughout

It's almost *tempting* to suggest using U+1D452 ("𝑒", or 'MATHEMATICAL ITALIC SMALL E'). That's exactly the right character to use in this context, but it's outside the Unicode basic multilingual plane, and I suspect that there would be rendering problems on at least some platforms.

Reply:
> I honestly don't think that ambiguity is likely to confuse people in practice.

Well, there was an example from the [issue thread](https://github.com/python/cpython/issues/101594#issuecomment-1418414210).

Reply:
> Well, there was an example from the https://github.com/python/cpython/issues/101594#issuecomment-1418414210.

Sorry, not convinced: you claim that Steven misinterpreted an `e` in the docstring as meaning `math.e`. Only @stevendaprano can tell us for sure, but I don't get that at all from his comment.